var/spack/repos/builtin/packages/py-darshan/package.py

```diff
@@ -18,7 +18,6 @@ class PyDarshan(PythonPackage):
 
     depends_on("python@3.6:", type=("build", "run"))
     depends_on("py-setuptools", type="build")
-    depends_on("py-importlib-resources", when="^python@3.6", type=("build", "run"))
```

Comment:
There actually turns out to be an error in this version constraint on our end. importlib-resources is required for all versions less than 3.9, not just version 3.6. Could we just change this to:

`when="^python@:3.8"`

That would correct the erroneous version constraint and support the changes attempted here related to EOL.

Reply:
Just to clarify, I think we want: `@:3.8` (not just 3.8)

Reply:
Also done